lib/errors.js

```diff
@@ -0,0 +1,24 @@
+import { ERROR_SCHEMA } from './constants';
+
+const scimTypes = {
+  400: 'badRequest',
+  401: 'unauthorized',
+  403: 'forbidden',
+  404: 'notFound',
+  422: 'unprocessableEntity',
+  500: 'internalServerError',
+  501: 'notImplemented',
+  502: 'badGateway',
+};
+
+export const ERROR_RES = (
```

Comment:
Using all uppercase for a function name is going to confuse a lot of developers.

```suggestion
export const errorRes = (
```

Reply:
Resolved